### PR TITLE
Add documentation regarding how to fix github issues with OpenDevin

### LINKED ISSUE #1828: [Documentation]: Documentation on fixing GitHub issues with OpenDevin
It would be nice to have better documentation on fixing github issues with OpenDevin. This should be added to the documentation in the `docs/` directory in the appropriate place.

A first draft of the required documentation is below between the lines:

------------

To fix an issue on OpenDevin, send a prompt like:
```
Please fix a github issue according to the following procedure:
1. Read the issue on https://github.com/OpenDevin/OpenDevin/issues/1611
2. Clone the repository and check out a new branch
3. Based on the instructions in the issue description, modify files to fix the issue
4. Push the resulting output to github using the GITHUB_TOKEN environment variable
5. Tell me the link that I need to go to to send a pull request
```

Before you run OpenDevin, you can do
```bash
export SANDBOX_ENV_GITHUB_TOKEN=XXX
```
where XXX is a github token that you created that has permissions to push to the OpenDevin repo. If you don’t have write permission to the OpenDevin repo, you might need to change that to:
```
4. Push the resulting output to my fork at https://github.com/USERNAME/OpenDevin/ using the GITHUB_TOKEN environment variable
```
where USERNAME is your github username.

------------

### PR DESCRIPTION
**Pull request by neubig**
Fixes #1828

------

This PR was entirely written by OpenDevin using GPT-4o. I had to send three corrections (in English) to fix mistakes, which you can see in the commits. Here were my prompts:

prompt 1
```
Please fix a github issue according to the following procedure:

Read the issue on https://github.com/OpenDevin/OpenDevin/issues/1828

Clone the repository and check out a new branch

Based on the instructions in the issue description, modify files to fix the issue

Push the resulting output to github using the GITHUB_TOKEN environment variable for authentication

Tell me the link that I need to go to to send a pull request
```

prompt 2
```
Thanks, this is a great start! But note that the user of OpenDevin does not need to perform these steps themselves, but rather needs to send a prompt to OpenDevin that describes these steps. Could you modify the documentation so that it says "To fix an issue GitHub using OpenDevin, send a prompt to OpenDevin asking it to follow these steps:"
```

prompt 3
```
It seems the now there are two lines "<h3>How to fix an issue on OpenDevin?</h3>", could you remove one?
```

prompt4
```
I'm getting an error about a missing close paragraph tag, can you fix it?
```

### CODE REVIEW

File: docs/src/pages/faq.tsx [hunk 1]
```diff
@@ -50,6 +50,26 @@ export default function FAQ() {
...
+          <ol>
+            <li>Read the issue on <a href="https://github.com/OpenDevin/OpenDevin/issues/1611">GitHub</a></li>
+            <li>Clone the repository and check out a new branch</li>
+            <li>Based on the instructions in the issue description, modify files to fix the issue</li>
+            <li>Push the resulting output to GitHub using the GITHUB_TOKEN environment variable</li>
+            <li>Tell me the link that I need to go to to send a pull request</li>
+          </ol>
+          Before you run OpenDevin, you can do:
+          <pre>
+            export SANDBOX_ENV_GITHUB_TOKEN=XXX
```
Comment (xingyaoww) [inline docs/src/pages/faq.tsx:65]: Is this actually implemented yet?

Comment (neubig) [reply docs/src/pages/faq.tsx:65]: Yes, it is.

Comment (xingyaoww) [reply docs/src/pages/faq.tsx:65]: `github_token` is the only one i can find -- not sure if `SANDBOX_ENV_GITHUB_TOKEN` will be actually set to `github_token` -- if so, i think i'm good with this one :D

https://github.com/OpenDevin/OpenDevin/blob/50c2141d2094c287f2bbd16d64ff482b96cd0e60/opendevin/core/config.py#L95

Comment (xingyaoww) [reply docs/src/pages/faq.tsx:65]: Bc i didn't actually see a prefix with `sandbox_env` in the `config.py`

Comment (enyst) [reply docs/src/pages/faq.tsx:65]: I believe env vars that start with SANDBOX_ENV_ are read from the env, bypassing the usual need to be in config.py
Update: found it https://github.com/OpenDevin/OpenDevin/blob/50c2141d2094c287f2bbd16d64ff482b96cd0e60/opendevin/runtime/sandbox.py#L16

Comment (xingyaoww) [reply docs/src/pages/faq.tsx:65]: @enyst ahhh got it! Thanks a ton!

**Review by enyst — APPROVED** [review]

**Pull request merged**